civic_europe/assets/scss/core/_basics.scss

```diff
@@ -35,6 +30,12 @@ h6 {
     line-height: 1.2;
     margin-bottom: 0.8 * $spacer;
     margin-top: 1.5 * $spacer;
+    font-weight: bold;
+}
+
+a,
+.btn-link {
+    font-weight: $demibold;
```

Comment:
is there a reason, you use demibold as the variable here, but change that below?